sdk/compute/Azure.ResourceManager.Compute/CHANGELOG.md

```diff
@@ -1,14 +1,20 @@
 # Release History
 
-## 1.6.0-beta.1 (Unreleased)
+## 1.6.0 (2024-08-23)
 
 ### Features Added
 
+- Added a new read-only property `LogicalSectorSize` to `DiskRestorePointData` class.
+- Added new properties `SkuProfile` and `ZonalPlatformFaultDomainAlignMode` to `VirtualMachineScaleSetData` class.
+- Added new properties `Zones`, `ZonalPlatformFaultDomainAlignMode`, and `SkuProfile` to `VirtualMachineScaleSetPatch` class.
+- Added a new property `ScheduledEventsPolicy` to `AvailabilitySetPatch` and `AvailabilitySetData` class.
+
 ### Breaking Changes
 
-### Bugs Fixed
+- `ExcludeExtensions` property in `ComputeSecurityPostureReference` is replaced by `ExcludeExtensionNames` property which takes in a list of strings.
+- `IsVmAgentPlatformUpdatesEnabled` property in `WindowsConfiguration` class is now read-only.
 
-### Other Changes
+## 1.6.0-beta.1 (Not Released)
```

Comment:
we should remove this line.

Reply:
i removed it but CI is failing now. which is why I added this line in the first place. 
Could you update it the way it needs to? 

We have 8/23 as release date that we want to meet